modules/layers/src/bitmap-layer/bitmap-layer-fragment.ts

```diff
@@ -40,6 +40,7 @@ uniform float desaturate;
 uniform vec4 transparentColor;
 uniform vec3 tintColor;
 uniform float opacity;
+uniform bool picking_uAttribute;
```

Comment:
Why do we have to declare this but not the other picking uniforms? is it not injected by the picking module? 

I would add some comment before the changed lines. I can't tell what `picking_uAttribute` means or what this is doing without looking up the picking module..

Reply:
It is not declared in the fragment shader by the picking module. I can move it into the module code itself.